### DISCUSSION TITLE
For 4 bit quantization, cogvlm-chat-v1.1, how much RAM and num of CPU do I need?

### DISCUSSION THREAD

**Discussion by LeslieWongCV** [THUDM/CogVLM#389] [Q&A] [answered]
Tried with:
32GB GRAM - V100
64 GB RAM
2 CPU
got Killed

**Comment by frode117** [comment]
[2024-02-29 01:25:24,679] [INFO] [RANK 0] global rank 0 is loading checkpoint /hy-tmp/cogvlm-chat-v1.1/1/mp_rank_00_model_states.pt
Killed

got killed too

24GB GRAM - 3090
32GB RAM
![issue1](https://github.com/THUDM/CogVLM/assets/94026101/21063eec-b6e8-4599-9ac6-207b30357a44)

**Comment by LeslieWongCV** [comment]
Any recommended CPU and RAM?

**Reply by zRzRzRzRzRzRzR** [reply, marked as answer]
这个应该是能跑得动吧（32G 需要 int4，建议使用A100 40G以上的）你CPU换好一点试试
